ubuntu/common/install_monitoring_tools.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+# Dependencies 
+pip3 install --upgrade pip
+python3 -m pip install ansible
+
+MONITOR_DIR=/opt/azurehpc/monitoring
```

Comment:
should we name it moneo itself?

Reply:
changed to tools directory as discussed.